docs/reference/glossary.md

```diff
@@ -483,203 +403,216 @@ Changes to the world are made by [workers](#worker). Each worker has a view onto
 
-The Unity ECS world is a new abstraction introduced by Unity to enable their Entity Component Systems. An ECS world contains a collection of ECS entities and systems to perform logic on these entities.
+## Unity ECS world
 
-In the GDK, ECS worlds (and everything in them) are an abstraction used to represent the part of the [SpatialOS world](#spatialos-world) that a worker has checked out. A worker is tightly coupled to its ECS world and defines which ECS entities belong to this world and which systems are run.
+<span style="display: block;margin-top: -20px;font-size: 10pt;"><i>Not to be confused with the [SpatialOS world](#spatialos-world).</i></span>
 
-> Related:
+An ECS world contains a collection of ECS entities and systems to perform logic on these entities.
+
+In the GDK, ECS worlds (and everything in them) are an abstraction used to represent the part of the [SpatialOS world](#spatialos-world) that a worker has checked out. A worker is coupled to its ECS world.
+
+> **Related:**
 >
 > * [Unity ECS documentation: World](https://docs.unity3d.com/Packages/com.unity.entities@0.0/manual/world.html)
 
-### Unity packages
+## Unity packages
 
-In Unity there are two [types of packages (Unity documentation)](https://docs.unity3d.com/Manual/AssetPackages.html):
+In Unity there are two [types of packages](https://docs.unity3d.com/Manual/AssetPackages.html):
 
-* Asset packages, available on the Unity Asset Store, which allow you to share and re-use Unity Projects and collections of Assets.
-* Unity packages, available through the Package Manager window (Unity documentation). You can import a wide range of Assets, including plugins directly into Unity with this type of package.
+* Asset packages, available on the Unity Asset Store, which allow you to share and re-use Unity Projects and collections of assets.
+* Unity packages, available through the Package Manager window. You can import a wide range of assets, including plugins directly into Unity with this type of package.
 
-In the GDK each feature module, and the core module, consists of Unity packages. They are located at `UnityGDK/workers/unity/Packages/`. This allows you to choose which feature modules you want to include in your game.
+In the GDK each feature module, and the core modules are Unity packages.
 
-### Unity Project
+## Unity Project
 
-> Not to be confused with [SpatialOS Project](#spatialos-project) or [project name](#project-name).
+<span style="display: block;margin-top: -20px;font-size: 10pt;"><i>Not to be confused with [SpatialOS Project](#spatialos-project) or [project name](#project-name).</i></span>
 
-A Unity project is the source code and assets of a SpatialOS game's Unity [workers](#worker). We often use this to refer to the directory that contains Unity code and assets, `<path to the GDK repo>/workers/unity` in this case.
+A Unity project is the source code and assets of a SpatialOS game's Unity [workers](#worker).
```

Comment:
```suggestion
A Unity project using the SpatialOS GDK contains the source code and assets of a SpatialOS game's Unity [workers](#worker).
```

otherwise it sounds like SpatialOS is always included in a Unity project ;)